.github/workflows/test-chart.yaml

```
@@ -237,11 +267,23 @@ jobs:
           timeout: 150
           max-restarts: 1
 
+      - name: "(Upgrade) Install PostgreSQL server chart"
+        if: matrix.setup-postgresql-args
+        run: |
+          . ./ci/common
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm install postgresql bitnami/postgresql ${{ matrix.setup-postgresql-args }}
```

I think `helm repo add` would also require `helm repo update`, which updates all declared helm repos. Better to do just `--repo=...` in the upgrade command to avoid a step like this I think.

```suggestion
          helm install postgresql postgresql --repo=https://charts.bitnami.com/bitnami ${{ matrix.setup-postgresql-args }}
```